framework/src/st_things/things_stack/things_stack.c

```diff
@@ -385,11 +389,45 @@ int things_start_stack(void)
 		THINGS_LOG_E(TAG, "es_conn_type_Unknown");
 		return 0;
 	}
-
 	THINGS_LOG_V(TAG, "Stack Start Success");
 	return 1;
 }
 
+int things_start_scanning_ap() 
+{
+	pthread_mutex_lock(&g_things_scan_ap_mutex);
+	g_scan_ap_continue_flag = true;
```

Comment:
I think we don't need g_scan_ap_continue_flag and g_things_scan_ap_mutex.
And hence we don't need things_stop_scanning_ap()